src/pod/personal-storage.ts

```diff
@@ -42,37 +51,20 @@ export class PersonalStorage {
    */
   async create(name: string): Promise<Pod> {
     assertActiveAccount(this.accountData)
-    name = name.trim()
-    assertPodName(name)
-    const podsInfo = await getPodsList(
+
+    const pod = await createPod(
       this.accountData.connection.bee,
-      prepareEthAddress(this.accountData.wallet!.address),
-      this.accountData.connection.options?.downloadOptions,
+      this.accountData.connection,
+      this.accountData.wallet!,
+      {
+        name,
+        index: 0,
```

Comment:
is the index always 0?

Reply:
no, later the index calculates with correct value